lib/job.js

```diff
@@ -243,12 +243,17 @@ function status(instance, job_id, job_execution_id, verbose, streamLog, openLogF
 }
 
 function buildParameters(job_params) {
```

Comment:
Fixed this method as the [OCAPI documentation](https://documentation.b2c.commercecloud.salesforce.com/DOC1/topic/com.demandware.dochelp/OCAPI/current/data/Documents/JobExecutionRequest.html#id515435266) states that the parameters should be in the following format:
```json
{
    "parameters": [{
        "name": "someName",
        "value": "someValue"
}]
}
```